besu/src/test/resources/everything_config.toml

```diff
@@ -16,7 +16,7 @@ node-private-key-file="./path/to/privateKey"
 pid-path="~/.pid"
 reorg-logging-threshold=0
 static-nodes-file="~/besudata/static-nodes.json"
-
+profile="NONE"
```

Comment:
The profile NONE does not exist in ProfileName. Should that not cause an error when the everything_config.toml is used? (e.g. in BesuCommandTest)

Reply:
that does not cause an error, in fact that appears several times in the `everything_config`